### PR TITLE
[CPM-42] associate creation date plan

### PR DESCRIPTION
**Pull request by Richardng12**
## Changes

- Added creation date to plans in student page

## Checklist

- [x] Have tests been written?
- [x] Is the UI consistent?

## Requests
- [ ] Can somebody check the styling/CSS?

Addresses [CPM-42]

Accidentally branched off another branch rather than pre-master, ignore Dalton and Nisarags commits pls

[CPM-42]: https://momentum-agile.atlassian.net/browse/CPM-42

### CODE REVIEW

File: client/src/pages/Plan/Plan.js [hunk 1]
```diff
@@ -197,13 +197,23 @@ const Plan = () => {
                     <Flex justify="center" width="100%">
                         <PlanButton
                             text="Remove Year"
-                            onClick={() => setNumYears(numYears - 1)}
+                            onClick={() => {
```
Comment (daltonlim) [inline client/src/pages/Plan/Plan.js:200]: undo Plan.js changes pls

Comment (Richardng12) [reply client/src/pages/Plan/Plan.js]: can u check now

Comment (daltonlim) [reply client/src/pages/Plan/Plan.js]: lgtm

**Review by daltonlim — APPROVED** [review]
Lgtm

**Review by wryao64 — APPROVED** [review]
LGTM

**Pull request merged**